docs/docs/getting-started/v3-migration.md

```diff
@@ -57,6 +58,7 @@ A number of changes were made to the configuration options passed to the `Chart`
 * Dataset options are now configured as `options[type].datasets` rather than `options.datasets[type]`
 * To override the platform class used in a chart instance, pass `platform: PlatformClass` in the config object. Note that the class should be passed, not an instance of the class.
 * `aspectRatio` defaults to 1 for doughnut, pie, polarArea, and radar charts
+* `TimeScale` does not read `t` from object data by default anymore. The default property is `x` or `y`, depending on the orientation.
```

Comment:
Can you still use `t` with a user-specified option? I haven't looked closely at this PR, but think that's the idea here. If so, it'd be useful to point to that option or the corresponding docs here

Reply:
Yes, added a link